firstvoices/backend/resources/songs.py

```diff
@@ -34,3 +35,11 @@ def before_import_row(self, row, row_number=None, **kwargs):
 
     def skip_row(self, instance, original, row, import_validation_errors=None):
         return not Song.objects.filter(id=row["parent_id"]).exists()
+
+    def after_import_row(self, row, row_result, row_number=None, **kwargs):
+        # If the book entry is a lyric and has notes, add them to the song
+        if Song.objects.filter(id=row["parent_id"]).exists() and row["notes"] != "":
+            song = Song.objects.get(id=row["parent_id"])
+            for note in row["notes"].split(self.array_sep):
```

Comment:
Optional, but the other book entry notes have some additional handling around whitespace which you could get by using (or taking from) `ArrayOfStringsWidget(sep=...).clean(x)` to generate the array of notes.